packages/main/test/samples/TextArea.sample.html

```diff
@@ -26,4 +26,16 @@ <h3>TextArea with Maximum Length</h3>
 	</xmp></pre>
 </section>
 
+<section>
+	<h3>TextArea with Label For It</h3>
+	<div class="snippet">
+		<ui5-label for="taWithLabelID">This should be in arial-label property of the text area</ui5-label>
```

Comment:
This is not an appropriate text for a label. Please think of a better Label and aim for a real sample, as this is the sample page and it is shown in the playground, not a test page